resources/kinesisanalytics-streams.go

```diff
@@ -0,0 +1,68 @@
+package resources
+
+import (
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/kinesisanalytics"
+)
+
+type KinesisAnalyticsStream struct {
+	svc             *kinesisanalytics.KinesisAnalytics
+	applicationName *string
+}
+
+func init() {
+	register("KinesisAnalyticsStream", ListKinesisAnalyticsStreams)
+}
+
+func ListKinesisAnalyticsStreams(sess *session.Session) ([]Resource, error) {
+	svc := kinesisanalytics.New(sess)
+	resources := []Resource{}
+	var lastApplicationName *string
+	params := &kinesisanalytics.ListApplicationsInput{
+		Limit: aws.Int64(25),
+	}
+
+	for {
+		output, err := svc.ListApplications(params)
+		if err != nil {
+			return nil, err
+		}
+
+		for _, applicationSummary := range output.ApplicationSummaries {
+			resources = append(resources, &KinesisAnalyticsStream{
+				svc:             svc,
+				applicationName: applicationSummary.ApplicationName,
+			})
+			lastApplicationName = applicationSummary.ApplicationName
+		}
+
+		if *output.HasMoreApplications == false {
+			break
+		}
+
+		params.ExclusiveStartApplicationName = lastApplicationName
+	}
+
+	return resources, nil
+}
+
+func (f *KinesisAnalyticsStream) Remove() error {
+
+	output, _ := f.svc.DescribeApplication(&kinesisanalytics.DescribeApplicationInput{
```

Comment:
This can easily fail. It's safer to check if an error occurred and return it in that case with:
```go
output, err := ...
if err != nil {
    return err
}
...
```